packages/bpk-component-dialog/src/BpkDialog.module.scss

```diff
@@ -32,11 +32,13 @@
 
   &__close-button {
     float: right;
-    margin: 0 0 tokens.bpk-spacing-base() tokens.bpk-spacing-base();
+    margin: 0 calc(tokens.bpk-spacing-sm() / 2) tokens.bpk-spacing-base()
```

Comment:
Linter required using `calc` or `math.div` for division

Reply:
Interesting, in another PR this wasn't needed and could be done via a variable: https://github.com/Skyscanner/backpack/pull/3320/files#r1542792398

Maybe we want to work on aligning and having a consistent approach throughout?

Reply:
Discussed in slack thread, discovered that the other PR's change does have the same linter warnings so changing this to a variable will not do anything to solve that problem